app/components/gcse_preview_component.html.erb

```diff
@@ -12,7 +12,7 @@
   </p>
 
   <% if course.additional_gcse_equivalencies %>
-    <%= simple_format(course.additional_gcse_equivalencies, class: "govuk-body") %>
+    <%= simple_format(markdown(course.additional_gcse_equivalencies), class: "govuk-body") %>
```

Comment:
could this be just`markdown`, or do we need simple format?
```suggestion
    <%= markdown(course.additional_gcse_equivalencies) %>
```

Reply:
Thanks, removed